src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 70px;
+  margin: 20px auto 0;
+}
+
+.search--small {
+  height: 42px;
+}
+
+.search__icon {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)